### PR TITLE
`vtctl OnlineDDL`: complete command set

### PR DESCRIPTION
**Pull request by shlomi-noach**
## Description

[vtctl](https://vitess.io/docs/17.0/reference/programs/vtctl/)'s `OnlineDDL` command now supports more operations, matching the complete `ALTER VITESS_MIGRATION ...` syntax.

It now supports these additional syntax:

```
vtctlclient -- OnlineDDL commerce cleanup 36422163_e32d_11ed_81c9_0a43f95f28a3
vtctlclient -- OnlineDDL commerce throttle 36422163_e32d_11ed_81c9_0a43f95f28a3
vtctlclient -- OnlineDDL commerce throttle all
vtctlclient -- OnlineDDL commerce throttle-all
vtctlclient -- OnlineDDL commerce unthrottle 36422163_e32d_11ed_81c9_0a43f95f28a3
vtctlclient -- OnlineDDL commerce unthrottle all
vtctlclient -- OnlineDDL commerce unthrottle-all
vtctlclient -- OnlineDDL commerce complete all
vtctlclient -- OnlineDDL commerce complete-all
vtctlclient -- OnlineDDL commerce launch 36422163_e32d_11ed_81c9_0a43f95f28a3
vtctlclient -- OnlineDDL commerce launch all
vtctlclient -- OnlineDDL commerce launch-all
```

At this time there's no compatible `vtctldclient` command - iterating in a future PR.
Documentation PR to follow.

## Related Issue(s)

- #6926 
- Documentation PR: https://github.com/vitessio/website/pull/1535

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [ ] Documentation was added or is not required

### CODE REVIEW

File: go/vt/vtctl/vtctl.go [hunk 1]
```diff
@@ -607,8 +607,19 @@ var commands = []commandGroup{
 					" \nvtctl OnlineDDL test_keyspace show running" +
 					" \nvtctl OnlineDDL test_keyspace show complete" +
 					" \nvtctl OnlineDDL test_keyspace show failed" +
+					" \nvtctl OnlineDDL test_keyspace cleanup 82fa54ac_e83e_11ea_96b7_f875a4d24e90" +
 					" \nvtctl OnlineDDL test_keyspace retry 82fa54ac_e83e_11ea_96b7_f875a4d24e90" +
-					" \nvtctl OnlineDDL test_keyspace cancel 82fa54ac_e83e_11ea_96b7_f875a4d24e90",
+					" \nvtctl OnlineDDL test_keyspace cancel 82fa54ac_e83e_11ea_96b7_f875a4d24e90" +
+					" \nvtctl OnlineDDL test_keyspace cancel-all" +
```
Comment (mattlord) [inline go/vt/vtctl/vtctl.go:613]: IMO `cancel all` is more intuitive, but it's a matter of taste and don't feel strongly about it. The benefit is that it's a single command/signature with the second argument being UUID/`all`.

**Review by mattlord — APPROVED** [review]

**Comment by mattlord** [comment]
LGTM. Can you please add a link to the docs PR in the description?

**Review by rohit-nayak-ps — APPROVED** [review]

File: go/vt/vtctl/vtctl.go (hunk 1 above)
Comment (shlomi-noach) [reply go/vt/vtctl/vtctl.go:613]: Agreed on `cancel all`. Right now this PR just makes `vtctldclient` compatible with `vtctlclient`. I will later on make `cancel all` available on both.

Comment (shlomi-noach) [reply go/vt/vtctl/vtctl.go:614]: `cancel all` (and likewise `complete all`, `throttle all`, etc.) are now supported. Refactored the code a bit to generalize the parsing of the command.

**Comment by shlomi-noach** [comment]
Documentation PR: https://github.com/vitessio/website/pull/1535

**Pull request merged**